modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
```

Comment:
There doesn't appear to be any changelog entry for the new `storageKey` option either. Is that expected as it's a public API?

Reply:
Sure, I'll add an entry for that.

Reply:
@HAFRMO you appear to have missed this...